src/applications/gi/components/profile/CalculatorForm.jsx

```diff
@@ -45,6 +45,25 @@ class CalculatorForm extends React.Component {
     return extensions;
   }
 
+  createExtensionOption = extension => {
+    const extensionOption = {
+      value: extension.zip,
+      label: '',
+    };
+
+    extensionOption.label = extension.institution;
+    if (extension.city && extension.state) {
+      extensionOption.label = `${extensionOption.label} (${extension.city}, ${
+        extension.state
+      })`;
+    } else if (extension.city) {
+      extensionOption.label = `${extensionOption.label} (${extension.city})`;
+    } else if (extension.state) {
+      extensionOption.label = `${extensionOption.label} (${extension.state})`;
+    }
```

Comment:
Could probably reduce the repetition of `extension.*` in this function by destructuring.

```
const { city, institution, state, zip } = extension;

...

extensionOption.label = `${institution} (${city}, ${state})`;
```

Reply:
Changes are added. Thanks for the feedback!